Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -1880,6 +1880,52 @@ public WilderMovingAverage WWMA(Symbol symbol, int period, Resolution? resolutio
             return wilderMovingAverage;
         }
 
+        /// <summary>
+        /// Creates a Wilder Swing Index (SI) indicator for the symbol.
+        /// The indicator will be automatically updated on the given resolution.
+        /// </summary>
+        /// <param name="symbol">The symbol whose SI we want</param>
+        /// <param name="limitMove">The maximum daily change in price for the SI</param>
+        /// <param name="resolution">The resolution</param>
+        /// <returns>The WilderSwingIndex for the given parameters</returns>
+        /// <remarks>SI for Wilder Swing Index</remarks>
+        public WilderSwingIndex SI(Symbol symbol, decimal limitMove, Resolution? resolution = Resolution.Daily)
+        {
+            var name = CreateIndicatorName(symbol, $"ASI", resolution);
```

Comment:
Can you please rename the indicator name "SI"

Reply:
Fixed in 87c8991